lib/server/clients.js

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Hapi = require('hapi');
+
+const AppError = require('../error');
+const auth = require('../auth');
+const config = require('../config').root();
+const logger = require('../logging')('server.clients');
+const hapiLogger = require('../logging')('server.hapi');
+const summary = require('../logging/summary');
+
+exports.create = function createServer() {
+  var isProd = config.env === 'prod';
+  var server = Hapi.createServer(
+    config.serverInternal.host,
+    config.serverInternal.port,
+    {
+      cors: true,
+      debug: false,
+      payload: {
+        maxBytes: 16384
+      },
+      security: {
+        hsts: {
+          maxAge: 15552000,
+          includeSubdomains: true
+        },
+        xframe: false,
+        xss: false,
+        noOpen: false,
+        noSniff: false
+      }
+    }
+  );
+
+
+
+  server.auth.scheme(auth.AUTH_SCHEME, auth.strategy);
+  server.auth.strategy(auth.AUTH_STRATEGY, auth.AUTH_SCHEME);
+
+  var routes = require('../routing').clients;
+  if (isProd) {
+    logger.info('prod', 'Disabling response schema validation');
+    routes.forEach(function(route) {
+      delete route.config.response;
+    });
+  }
+
+  // require json by default
```

Comment:
Comment is out-of-sync with `allow: ['application/json', 'application/x-www-form-urlencoded']`